tests/cli/test_main_run.py

```diff
@@ -48,7 +52,10 @@ def test_run_uncaptured(tmp_env: TmpEnvFixture, conda_cli: CondaCLIFixture):
     with tmp_env() as prefix:
         random_text = uuid.uuid4().hex
         stdout, stderr, err = conda_cli(
-            "run", "--prefix", prefix, "--no-capture-output", "echo", random_text
+            "run",
+            f"--prefix={prefix}",
+            "--no-capture-output",
+            *("echo", random_text),
```

Comment:
I've seen this format a few times now, do we need to pack and unpack these tuples here?

```suggestion
            "echo",
            random_text,
```

Reply:
no we don't have to do this, I've been doing this to indicate that the arguments "belong together"